menas/ui/components/dataset/datasetMain.controller.js

```diff
@@ -145,6 +145,12 @@ sap.ui.controller("components.dataset.datasetMain", {
                 "Dataset name '" + oDataset.name + "' already exists. Choose a different name.");
             isOk = false;
         }
+        if (/\W/.test(oDataset.name)) {
```

Comment:
Maybe we should create a "generic" name validation function... Same will apply for other controllers too

Reply:
Let's create the _generic_ function only when needed and **if** the validation is the same 🤔

Reply:
The point here is that, they are the same validations (checking the name of the entity, dataset/schema/mapping table), rather than repeat the same function (with the odd JS regex syntax) in three places, we can reduce code duplication.

Reply:
resolved.